### ISSUE TITLE
language.js unnecessary

### ISSUE THREAD

**Issue by tinagrit** [tinagrit/mumaths#1]
Different pages have their own javascript files that implement languages. ie: `calculation.js ` has `innerHTML` functions that refer to the global `lang` variable. `language.js` only applies to the global navigation. It only has two lines for now but I don't see any use of it in the future. It is a better idea to move it into `ui.js` which would make more sense as `ui.js` is applied globally and already deals with Typer and global functions.

_**TLDR**: Move codes from `assets/languages/language.js` into `assets/js/ui.js` and delete `assets/languages/language.js`_
